packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/middleware/plugins/personalize.ts

```diff
@@ -29,9 +40,14 @@ class PersonalizePlugin implements MiddlewarePlugin {
             parseInt(process.env.PERSONALIZE_MIDDLEWARE_CDP_TIMEOUT)) ||
           250,
       },
-      disabled: () => {
-        return false;
-      },
+      // This function determines if the middleware should be turned off.
+      // This would typically be based on your cookie consent management solution of choice.
+      // By default it is always enabled.
+      disabled: () => false,
+      // This function is used to determine if route should be excluded from personalization.
```

Comment:
Minor adjustment for consistency with "disabled".
```suggestion
      // This function determines if a route should be excluded from personalization.
```